src/redux/migrations.ts

```diff
@@ -892,4 +892,11 @@ export const migrations = {
     ...state,
     app: _.omit(state.app, 'celoWithdrawalEnabledInExchange'),
   }),
+  93: (state: any) => ({
+    ...state,
+    fiatConnect: {
+      ...state.fiatConnect,
+      transfer: null,
```

Comment:
I am not sure actually if we need this migration, since we reset transfer to `null` on `REHYDRATE` and all the schema changes are under transfer. I can remove this if anyone can confirm this is not required.

Reply:
i don't think this migration is needed for the rehydrate reason you mentioned. but at the same time it doesn't hurt to exist if you want to leave it. the main purpose for migrations is to prevent breaking changes on app update - migrations which deal with removing properties are not strictly required but do become important if we were ever going to remove a property then bring it back with a different meaning. e.g. if in version 0 we had property A and in version 2 removed property A but brought it back in version 10 with a different meaning, a user who is upgrading from version 0 to 10 would have a property A which could potentially cause problems. seems unlikely that you are going to run into this edge case with this redux property though